sdk/storage/Azure.Storage.Blobs/tests/BlobClientTests.cs

```diff
@@ -699,5 +699,94 @@ public async Task UploadAsync_OverwritesDeliberately_Path()
             }
         }
         #endregion Upload
+
+        [Test]
+        [LiveOnly] // Don't want to record 1000 downloads of the same bytes
+        public async Task Perf_SmallBlobs()
```

Comment:
I don't see what this test is automatically checking. What is the fail condition?

Reply:
It's for getting perf traces.  I'll add an `[Explicit]` instead.